### PR TITLE
Renames possibleValues to limitedToValues. Closes #58.

### LINKED ISSUE #58: Rename property possibleValues to limitedToValues in Variables
It is a bit unclear that `possibleValues` are constraints.  `limitedToValues` will help clear it up.

### PR DESCRIPTION
**Pull request by chriscox**

### CODE REVIEW

File: src/core/Remixer.ts [hunk 1]
```diff
@@ -168,60 +168,60 @@ class Remixer {
...
   /**
    * Adds a string Variable to array of variables with optional callback
-   * @param  {string}              key            The key of the Variable.
-   * @param  {string}              defaultValue   The initial default value of the variable.
-   * @param  {string[]}            possibleValues The optional array of available items for the variable.
-   * @param  {IVariableCallback}   callback       The callback method to be invoked
-   *                                              when the Variable is updated.
+   * @param  {string}              key             The key of the Variable.
+   * @param  {string}              defaultValue    The initial default value of the variable.
+   * @param  {string[]}            limitedToValues The optional array of available items for the variable.
```
Comment (chuga) [inline src/core/Remixer.ts]: The description could be improved, what about: "When set, the variable can only be set to one of these values", or "The optional array of allowed values"?

File: src/core/variables/Variable.ts [hunk 2]
```diff
@@ -33,12 +33,13 @@ export interface IVariableParams {
 }
 
 /**
- * Interface for a class that represents a type a Variable with possible values.
+ * Interface for a class that represents a type of Variable with an array of
```
Comment (chuga) [inline src/core/variables/Variable.ts]: "... represents a type of Variable that can only be set to predefined values"

**Review by chuga — APPROVED** [review]

File: src/core/Remixer.ts (hunk 1 above)
Comment (chriscox) [reply src/core/Remixer.ts]: done.

File: src/core/variables/Variable.ts (hunk 2 above)
Comment (chriscox) [reply src/core/variables/Variable.ts]: done.

**Pull request merged**